src/main/java/com/example/weatherwaveapi/model/response/weatherapi/weather/type/Wind.java

```diff
@@ -0,0 +1,10 @@
+package com.example.weatherwaveapi.model.response.weatherapi.weather.type;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record Wind(
+        double speed,
```

Comment:
you need to use wrappers for primitives, otherwise if one of the parameters is null, you will get NullPointerException